Updated documentation comments

JerelJr opened the pull request:
In reference to #3.

- Moved function description comments to header files
- Corrected `tcp_network_manager` description; return type is bool not void
- Regenerated documentation

VedantParanjape commented:
Hey Thanks for the PR. Can you remove the generated html files? I will handle that on end!

JerelJr commented:
Can do!

VedantParanjape commented:
Hey there are still changes in docs folder.

JerelJr commented:
My mistake. I thought you only wanted the HTML files removed

VedantParanjape commented:
Hey, I think you messed up the commit, you need to roll back the commit and then just add the header changes to the commit. Now it seems you deleted the complete doc folder in your repo.

JerelJr commented:
Ah my mistake. Will fix.

JerelJr committed:
Updated documentation comments

- Moved function description comments to header files
- Corrected tcp_network_manager function description

VedantParanjape commented on include/utils.h:
```diff
@@ -7,10 +7,18 @@
 }
 #endif
 
-#endif
```
can you add back the new line?

VedantParanjape commented on include/wifi_logger.h:
```diff
@@ -23,19 +23,67 @@ extern "C" {
 #endif
 
 #endif
-
```
this as well.

VedantParanjape commented on utils.cpp:
```diff
@@ -19,6 +11,6 @@ char* generate_log_message_timestamp(uint log_level, uint32_t timestamp, char* l
-    
+
     return c_log_string;
-}
```
this as well.

JerelJr commented:
Just for clarity, you want me to add a new line to the end of each of those files, correct?

VedantParanjape commented:
> Just for clarity, you want me to add a new line to the end of each of those files, correct?

yeah, correct, undo the changes basically.

VedantParanjape commented:
ping

JerelJr committed:
Appended newline to some files

JerelJr commented:
Sorry for the delay. I added the newlines.
